modules/wallet.go

```diff
@@ -126,7 +158,7 @@ type Wallet interface {
 
 	// SendSiagSiafunds sends siafunds to another address. The siacoins stored
 	// in the siafunds are sent to an address in the wallet.
-	SendSiagSiafunds(amount types.Currency, dest types.UnlockHash, keyfiles []string) (types.Transaction, error)
+	SendSiagSiafunds(amount types.Currency, dest types.UnlockHash, keyfiles []string) ([]types.Transaction, error)
```

Comment:
I feel like the wallet shouldn't need to know about `siag`. Not a big deal though.